src/main/java/data/workouts/WorkoutList.java

```diff
@@ -4,10 +4,13 @@
 import data.exercises.InvalidExerciseException;
 
 import java.util.ArrayList;
+import java.util.Scanner;
 
 public class WorkoutList {
     ExerciseList exerciseList;
     ArrayList<Workout> workoutsList = new ArrayList<>();
+    public static final int MAX_DISPLAY = 10;
+
 
```

Comment:
Perhaps this extra empty line can be removed?

Reply:
I agree.